<launch>
  <arg name="GROUP" default="/robot_0"/>
  <arg name="ROBOT_NAME" default="turtlebot3"/>
  <!--megarover turtlebot3 beego-->
  <arg name="IS_SIMULATOR" default="true"/>
  <arg name="USE_RVIZ" default="true"/>
  <arg name="PUBLISH_COMMAND" default="true"/>

  <group ns="$(arg GROUP)">

    <include file="$(find potbot)/launch/PathPlanning.launch">
      <arg name="USE_RVIZ" value="$(arg USE_RVIZ)"/>
    </include>

  </group>

</launch>
